pkg/operator/sync.go

```diff
@@ -267,6 +267,7 @@ func (optr *Operator) syncRenderConfig(_ *renderConfig) error {
 	spec.RootCAData = bundle
 	spec.PullSecret = &corev1.ObjectReference{Namespace: "openshift-config", Name: "pull-secret"}
 	spec.OSImageURL = imgs.MachineOSContent
+	spec.ReleaseImage = imgs.ReleaseImage
```

Comment:
following offline discussions - this isn't needed as we'll have this from the CVO crd